.github/workflows/ci-e2e-all.yml

```diff
@@ -36,7 +36,22 @@ jobs:
   opensearch:
     uses: ./.github/workflows/ci-e2e-opensearch.yml
 
-
-
+  metrics-collection:
```

Comment:
scraping needs to happen in each of the e2e tests. There's going to be no Jaeger containers running once this job is running.